### ISSUE TITLE
separate trainingsset into cv-set and test-set 

### ISSUE THREAD

**Issue by BennyBlanco22** [Microsoft/CNTK#1689]
Usually the database for a machine learning  algorithm exists of a 60% training set, 20% validation set and 20% testing set. Is there a way to automatically separate the training set using brainscript? The current approach would be to separate the training manually into those sets. Is there a requirement for this feature? If so I could propose a code which extends cntk to do so. The result could be an easy way to individually split the database by random assignments of data to each set.

**Comment by cha-zhang** [comment]
Thanks for the suggestion. We don't have such functionality. It should be easy to write in Python, though.
